### PR TITLE
fix(sidebar, mob footer): set proper active state for All Daos and My Daos links

### LINKED ISSUE #356: [Sidebar, App Footer Mobile] Proper state of All Daos and My Daos links
Keep state of All Daos and My Daos links "active" if user is surfing through correspondent daos.

sidebar

app-footer-mobile

### PR DESCRIPTION
**Pull request by unholywhite**

### CODE REVIEW

File: astro_2.0/components/navigation/MobileNav/MobileNav.tsx [hunk 1]
```diff
@@ -14,8 +15,22 @@ import { NavButton } from 'astro_2.0/components/navigation/NavButton';
...
 export const MobileNav: VFC = () => {
+  const [myDaosIds, setMyDaosIds] = useState<string[]>([]);
   const { accountId } = useAuthContext();
 
+  useEffect(() => {
+    Promise.allSettled([SputnikHttpService.getAccountDaos(accountId)]).then(
+      res => {
+        const [accountDaosRes] = res;
+        const accountDaos =
+          accountDaosRes.status === 'fulfilled' ? accountDaosRes.value : [];
```
Comment (VERZUOL1) [inline astro_2.0/components/navigation/MobileNav/MobileNav.tsx]: Do we really need Promise.allSettled here? There is only one request here.

File: astro_2.0/components/navigation/NavButton/NavButton.tsx [hunk 2]
```diff
@@ -40,7 +44,15 @@ export const NavButton: VFC<NavButtonProps> = props => {
 
   const [showLoader, setShowLoader] = useState(false);
 
-  const isActive = useIsHrefActive(href);
+  const { asPath } = router;
+  const pathParts = asPath.split('/').filter(item => item !== '');
+  const daoId = pathParts[0] === 'dao' ? pathParts[1] : '';
```
Comment (VERZUOL1) [inline astro_2.0/components/navigation/NavButton/NavButton.tsx]: We can get active dao id from url just from router.query object.
const daoId = router.query.dao

**Review by VERZUOL1 — CHANGES_REQUESTED** [review]

File: astro_2.0/components/navigation/MobileNav/MobileNav.tsx (hunk 1 above)
Comment (unholywhite) [reply astro_2.0/components/navigation/MobileNav/MobileNav.tsx]: fixed

File: astro_2.0/components/navigation/NavButton/NavButton.tsx (hunk 2 above)
Comment (unholywhite) [reply astro_2.0/components/navigation/NavButton/NavButton.tsx]: fixed

**Review by VERZUOL1 — APPROVED** [review]

**Pull request merged**